baybe/campaign.py

```diff
@@ -119,19 +112,21 @@ def from_config(cls, config_json: str) -> Campaign:
         Returns:
             The constructed campaign.
         """
+        from baybe.deprecation import compatibilize_config
```

Comment:
why not top level import?

Reply:
Classic: circular import. For this particular one I wouldn't even care too much since it's deprecation code anyway. However, perhaps we can at some point do one iteration and see if we can remove some of the delayed imports at other places.... For now, I'll resolve though